src/flash-source-buffer.js

```diff
@@ -169,6 +169,12 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
     };
 
     chunkInData();
+
+    // fire loaded metadata on first segment append
+    if (!this.firstAppendHappened) {
```

Comment:
The only way I could think of doing it was firing it when we append the encoded flv header. Unfortunately it was throwing off some tests. We don't have anything keeping track of the number of segments we have added to the buffer, and I can't check if the sourceBuffer is empty because of the already appended flv header. Definitely open to suggestions on a better way to do it, but currently haven't had much luck.